_source/_posts/2019-05-30-vue-pwa-typescript.md

```diff
@@ -308,7 +315,7 @@ This component simply shows the book's details obtained from the route's query p
 
 When the application was generated by the `vue` command line tool, it also created a `HelloWorld` component at `src/components/HelloWorld.vue`. This is not needed in the application, so you can delete it. If you delete this file, you'll need to delete references to it in `src/views/Home.vue` as well.
 
-In order for a sub-component to be shown, it must be registered with the router. Open `src/router.ts` and replace it with the code below.
+In order for a sub-component to be shown, it must be registered with the router. Open `src/router/index.ts` and replace it with the code below.
```

Comment:
`src/router.ts` is the correct filename: https://github.com/oktadeveloper/okta-vue-books-pwa-example/blob/master/src/router.ts

Reply:
that's not how the latest version of the cli builds it

Reply:
In that case, the example needs an update too. :)

Reply:
done!